### PR TITLE
[Fixtures] Payment/shipping method fixtures translations + examples

### LINKED ISSUE #10775: [Fixtures] Shipping/Payment method fixtures have no ability to specify translations
**Sylius version affected**: 1.6

**Description**  
When we compare `TaxonFixture` with `ShippingMethodFixture`/`PaymentMethodFixture` we can see that first one have ability to specify `translations`, but other two - not.

**Possible Solution**  
Add that as a bugfix :)

I will provide PR if you OK with that (I need that at 1.6).

### PR DESCRIPTION
**Pull request by igormukhingmailcom**
| Q               | A
| --------------- | -----
| Branch?         | 1.6
| Bug fix?        | maybe :)
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #10775
| License         | MIT

### CODE REVIEW

**Comment by Zales0123** [comment]
Hello Igor! :) It's a nice concept, but I would personally avoid traits. They do not provide a lot of benefits, but also make the implementation less clear and readable. cc @pamil

**Comment by igormukhingmailcom** [comment]
Ok, will refactor then

**Comment by Zales0123** [comment]
Hello @igormukhingmailcom, any updates about this PR? :)

**Comment by igormukhingmailcom** [comment]
Ah, will add fixes soon, @Zales0123

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/TranslatableExampleFactoryTrait.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{
+    /** @var OptionsResolver */
+    private $optionsResolver;
+
+    abstract protected function createTranslation(TranslatableInterface $translatable, string $localeCode, array $options = []): void;
+
+    protected function configureTranslationsOptions(OptionsResolver $resolver): void
+    {
+        $resolver
+            ->setDefault('translations', [])
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/TranslatableExampleFactoryTrait.php]: I'm wondering if we shouldn't provide some way to define what translations we can expect to have here. Something similar to `createTranslation`. On the other hand, your proposal is identical to the solution we have already

**Pull request closed without merging**